src/test/scala/lenthall/config/ScalaConfigSpec.scala

```diff
@@ -23,6 +25,7 @@ class ScalaConfigSpec extends FlatSpec with Matchers {
   it should "return the config value as options when present" in {
     exampleConfig.getConfigOption("configVal") should be('defined)
     exampleConfig.getStringOption("stringVal") should be(Some("string"))
+    exampleConfig.getPathOption("stringVal") should be(Some(Paths.get("string")))
```

Comment:
FWIW when I find myself typing `getStringOption map { Paths.get(_) }` repeatedly (as I did) I put it into a function. And as I said earlier, I figured someone would say "put it in lenthall"